javascriptv3/example_code/personalize/src/personalize_createCampaign.js

```diff
@@ -0,0 +1,49 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with the AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3.
+
+Purpose:
+createCampaign.js demonstrates how to create a campaign with Amazon Personalize. A campaign
+is a deployed solution version (trained model) with provisioned dedicated 
+transaction capacity for creating real-time recommendations for your application users.
+See https://docs.aws.amazon.com/personalize/latest/dg/API_CreateCampaign.html.
```

Comment:
For more information, see https://docs.aws.amazon.com/personalize/latest/dg/API_CreateCampaign.html.